tests/testthat/test-compute_kidney.R

```diff
@@ -71,15 +71,15 @@ test_that("compute_egfr Test 7: CKD-EPI calculated on input data", {
   input <- tibble::tribble(
     ~STUDYID, ~USUBJID, ~AGE, ~SEX, ~RACE, ~WTBL, ~CREATBL, ~CREATBLU,
     "P01", "P01-1001", 55, "M", "WHITE", 90.7, 96.3, "umol/L",
-    "P01", "P01-1002", 52, "F", "BLACK OR AFRICAN AMERICAN", 68, 70, "umol/L",
+    "P01", "P01-1002", 52, NA_character_, "BLACK OR AFRICAN AMERICAN", 68, 70, "umol/L",
```

Comment:
I think a new records should be added for `SEX = NA`. Otherwise, the case black female is no longer covered.